src/hooks/useSafeTransactionStatus.ts

```diff
@@ -0,0 +1,53 @@
+import { defineMessages } from 'react-intl';
+
+import { ETHEREUM_NETWORK } from '~constants';
+import { useGetSafeTransactionStatusQuery } from '~gql';
+import { ColonyAction } from '~types';
+
+export enum TRANSACTION_STATUS {
+  COMPLETED = 'Completed',
+  ACTION_NEEDED = 'Action neeeded',
```

Comment:
typo :slightly_smiling_face: 
```suggestion
  ACTION_NEEDED = 'Action needed',
```